maintainers/maintainer-list.nix

```diff
@@ -64,6 +64,12 @@
     githubId = 791309;
     name = "Jan Hrnko";
   };
+  _3699n = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add _3699n``.

Reply:
Okay I can do that. I thought you asked me to squash it?

Reply:
I meant to squash review changes into the update commit.